src/Extensions/Hosting.Services.Web/Middlewares/ObsoleteCorrelationHeadersMiddleware.cs

```diff
@@ -20,12 +20,9 @@ namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
 	/// This middleware should be removed after all services moved to use <see cref="Activity"/>
 	/// </remarks>
 	[Obsolete("Use it only if you need to comunicate with services that use old correlation", false)]
-	internal class ObsoleteCorrelationHeadersMiddleware
+	internal class ObsoleteCorrelationHeadersMiddleware : IMiddleware
 	{
-		/// <summary>
-		/// Invoke middleware
-		/// </summary>
-		public Task InvokeAsync(HttpContext context)
+		Task IMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
```

Comment:
Can't it be simplified to `InvokeAsync`?

Reply:
It can, I've decided to use explicit interface implementation to emphasize the fact that the method should not be called from the instance of a class.